content/post/mnist-example.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
+tags = [
+    "wallaroo",
+    "machine learning",
+    "python",
+    "example",
+    "tutorial"
+]
+categories = [
+    "Machine Learning"
+]
+description = "Creating an inference pipeline with MNIST"
+author = "amosca"
++++
+
+Nowadays, many applications with streaming data are either applying machine learning or have a very good use case for it. In this example, we will explore how we can build a sample pipeline to classify images from the [MNIST dataset](ihttp://yann.lecun.com/exdb/mnist/), using PCA and a Logistic Regression in scikit-learn. As a prerequisite, you should be familiar with some notions of machine learning, and some notions about Wallaroo pipelines.
+Everything has been implemented in the [current version of Wallaroo (0.4.0)](https://github.com/WallarooLabs/wallaroo/tree/0.4.0). The full code can be found on [GitHub](https://github.com/WallarooLabs/wallaroo_blog_examples/tree/master/sklearn-example). If you have any technical questions that this post didn't answer, or if you have any suggestions, please get in touch at [hello@WallarooLabs.com](mailto:hello@WallarooLabs.com), via [our mailing list](https://groups.io/g/wallaroo) or [our IRC channel](https://webchat.freenode.net/?channels=#wallaroo).
+
+
+## The MNIST dataset
+
+The MNIST dataset is a set of 60000 black and white images, of size 28 x 28 pixels, contaning hand-written digits from 0 to 9. Each of these images has been classified and it is often used as a benchmark for computer vision and machine learning. The pixels are real valued, between 0 (completely black) and 1 (completely white).
+
+## The model
+
+One of the simplest models for classifying digits is a [Logistic Regression](https://en.wikipedia.org/wiki/Logistic_regression) on the numeric values of each pixel. An improvement to this simple model is to add a [PCA](https://en.wikipedia.org/wiki/Principal_component_analysis) preprocessing that presents the transformation with the most information content as an input to the classifier.
```

Comment:
It would be great if we can write out `Principal Component Analysis (PCA) so people who aren't as familiar with ML know what PCA stands for.

I don't know what the goal for this post is (or who the audience is), but it would be great if you are able to include a 1 or 2 line explanation of how LR and/or PCA do what they do and why classification is important.

Also, this section is talking about `classifying digits` but the previous section said `Each of these images has been classified`. Why would we need to classify digits if the images are already classified?